webpay/base/templates/base.html

```diff
@@ -84,10 +84,5 @@ <h2 class="heading"></h2>
       <div class="throbber"></div>
       <span class="txt"></span>
     </div>
-
-    {% block site_js %}
-      <script src="{{ settings.BROWSERID_JS_URL }}"></script>
-      {{ js('pay') }}
-    {% endblock %}
```

Comment:
That'll probably break tests so I thought we should leave that. It'll be easier to see any regression doing this piece by piece.